app/javascript/application.js

```diff
@@ -1,3 +1,16 @@
-// Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function App() {
+  return (
+  <dib>
+    <h1>Hello World!</h1>
+    <p>This is react-rails project</p>
+  </dib>
+  );
+}
+
```

Comment:
- [ ] Nice work following the tutorial provided :clap: However the appropriate `App` component would be the one you created in the `components/App.js` . Therefore I suggest you remove this `App()` function and import the component `App`